src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipeflatlands.java

```diff
@@ -24,7 +25,21 @@ public boolean run(final CommandSender sender, Player playerSender, Command cmd,
             player.kickPlayer("Server is going offline for flatlands wipe, come back in a few minutes.");
         }
 
-        server.shutdown();
+        if (!plugin.cpb.isEnabled())
```

Comment:
If I'm reading this correctly, you want me to add a config value to state rather or not to use the bridge even if CoreProtect is installed and enabled?